catchpoint/scripts/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM --platform=linux/arm64 golang:latest as builder
+
+WORKDIR /app
+
+
+RUN apt-get update && apt-get install -y git curl jq
+RUN git clone https://github.com/grafana/catchpoint-prometheus-exporter.git .
+RUN git fetch origin +refs/pull/*/head:refs/remotes/origin/pr/*
+# undo the requirement for a PR number after pr is merged
+RUN PR_NUM=$(curl -s https://api.github.com/repos/grafana/catchpoint-prometheus-exporter/pulls | jq -r '.[0].number') \
+    && git checkout origin/pr/$PR_NUM
```

Comment:
Can you please make this PR a draft until this is resolved, would prefer we don't accidentally merge this in